gradle.properties

```diff
@@ -1,17 +1,41 @@
 # suppress inspection "UnusedProperty" for whole file
 
 group=org.pkl-lang
-version=0.26.0
+version=0.27.0-kotlin-1.9.22
+
+# bytecode targeting & compiler settings
+strict=false
+javaTarget=11
+javaEntrypointTarget=11
+javaToolchainTarget=21
+kotlinTarget=1.9
+kotlinBeta=false
+
+# build settings
+lockDependencies=false
+xmlReporting=false
+sarifReporting=false
+htmlReporting=false
+autofixDetekt=false
+remoteCache=false
+cachePush=false
+
+# kotlin settings
+kotlin.stdlib.default.dependency=false
+
+# gradle settings
+org.gradle.caching=true
+org.gradle.configuration-cache=false
+org.gradle.configuration-cache-problems=warn
+org.gradle.dependency-verification=lenient
+org.gradle.parallel=true
```

Comment:
Adjustable build settings in the root `gradle.properties`. Dependency Verification is left in `LENIENT` mode for now so that configurations can be built and then sealed.